test cases/frameworks/17 mpi/main.cpp

```diff
@@ -1,11 +1,17 @@
 #include <mpi.h>
-#include <stdio.h>
+#include <iostream>
 
 int main(int argc, char **argv)
 {
+    int L;
+
+    char version[MPI_MAX_LIBRARY_VERSION_STRING];
+    MPI_Get_library_version(version, &L);
+    std::cout << version << std::endl;
+
     MPI::Init(argc, argv);
     if (!MPI::Is_initialized()) {
-        printf("MPI did not initialize!\n");
+        std::cerr << "MPI did not initialize!\n";
```

Comment:
presumably you should be using `std::endl` here as well.

Reply:
`std::endl` usage is considered bad practice though, since it forces a flush to output (it's more chars to type as well =)).
Not that it matters in this case (being an error message and all that).

Reply:
I suppose here a flush could be beneficial, considering parallel tests, the flush would make it a bit more likely the stderr message is proximate to other same-test messages.